src/amchart/Scatter.js

```diff
@@ -20,6 +20,9 @@
 
     Scatter.prototype.publish("paletteID", "default", "set", "Palette ID", Scatter.prototype._palette.switch(), {tags:["Basic","Shared"]});
 
+    Scatter.prototype.publish("chartMode", "scatter", "set", "Bullet Type", ["scatter", "bubble"],{tags:["Basic"]});
```

Comment:
**chartMode** is ambiguous. Something like **scatterType** would be more appropriate.